src/pybind/mgr/dashboard_v2/services/ceph_service.py

```diff
@@ -53,8 +133,96 @@ def get_service(cls, service_name, service_id):
 
     @classmethod
     def get_pool_list(cls, application=None):
-        osd_map = cls.mgr.get('osd_map')
-        if not application:
-            return osd_map['pools']
-        return [pool for pool in osd_map['pools']
-                if application in pool.get('application_metadata', {})]
+        return _get_pool_list(cls.mgr, application)
+
+
+class CephDaemonMixin(object):
+    @classmethod
+    def get_daemons(cls):
+        return _get_daemons(cls.mgr)
+
+    @classmethod
+    def get_daemon_health(cls, daemon):
+        return _get_daemon_health(daemon)
+
+
+class CephPoolMixin(object):
```

Comment:
This mixin class should be merged with the mixin class above, and renamed to `CephRbdMirrorMixin` as suggested in my previous comment.